codalab/model/bundle_model.py

```diff
@@ -774,7 +774,7 @@ def do_update(connection):
     def get_bundle_state(self, uuid):
         result_dict = self.get_bundle_states([uuid])
         if uuid not in result_dict:
-            raise UsageError('Could not find bundle with uuid %s' % uuid)
+            raise NotFoundError('Could not find bundle with uuid %s' % uuid)
```

Comment:
I know NotFoundError is logical, but a lot of the CLI code relies on catching UsageErrors upstream, so I'd change this back for now.